Documentation/avatar/index.html

```diff
@@ -6,9 +6,290 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="author" content="Aurobindo Gupta" />
     <meta name="description" content="Component library content - Avatar " />
-    <title>Alert</title>
+    <link rel="stylesheet" href="styles.css" />
+    <title>Avatar</title>
   </head>
   <body>
-    <h1>ALERT</h1>
+    <!-- ................BASE CONTAINER............. -->
+    <div class="baseContainer">
+      <!-- ................NAV BAR............. -->
+
+      <nav class="navBar">
+        <div class="heroContainer">
+          <img
+            src="/Utils/images/panda sulk2.jpg"
+            class="navbarHeroImg"
+            onclick="window.location.href='./Components/index.html'"
+          />
+          <p class="heroTitle">PandaUI</p>
+        </div>
+        <ul class="navLinks">
```

Comment:
add some padding to the link.
It is changes the place of other links while you hover on any of them in the navigation bar.